pages/upload.js

```diff
@@ -145,8 +162,8 @@ class FilePicker extends React.Component {
           <span>
           {
             this.state.active
-              ? <h2>Drop file to upload</h2>
-              : <h2>Drag file here</h2>
+                ? <h2><span className="underline">Drop</span> file to upload</h2>
+                : <h2><span  className="underline">Drag</span> a file or <span className="underline">paste</span> an image here</h2>
```

Comment:
Screenshot? What's this look like stylistically? (I've been roughly trying to match slack styles, so this might be a departure from it)